packages/lesswrong/lib/collections/users/custom_fields.ts

```diff
@@ -1788,6 +1788,36 @@ addFieldsDict(Users, {
     },
     group: formGroups.socialMedia
   },
+  postingDisabled: {
+    type: Boolean,
+    optional: true,
+    canRead: ['members'],
+    canUpdate: ['sunshineRegiment', 'admins'],
+    canCreate: ['sunshineRegiment', 'admins'],
+    control: 'checkbox',
+    group: formGroups.disabledPrivileges,
+    order: 69,
+  },
+  commentingDisabled: {
```

Comment:
I do think it's quite plausible that "only allow people to comment on their shortform" is a setting we'd want. (This might actually be the main way we want mostly-banned-users to continue to contribute, since it neither clogs up Latest Posts or the comment threads of other posts)

Right now this PR isn't top priority so I'd prefer to ship something close to the current version for now, but I agree that's a pretty reasonable tweak to it.